deploy/Datadog.Trace.ClrProfiler.WindowsInstaller/Datadog.Trace.ClrProfiler.WindowsInstaller.wixproj

```diff
@@ -14,25 +14,17 @@
     <IntermediateOutputPath>obj\$(Configuration)\$(Platform)\</IntermediateOutputPath>
     <SuppressPdbOutput>True</SuppressPdbOutput>
     <DefineSolutionProperties>false</DefineSolutionProperties>
-    <!-- Custom InstallerVersion property for the generated installer,
-         value is set to build constant InstallerVersion below and
-         accessed as $(var.InstallerVersion) from WiX files.
-         This is the default value if one is not specified when building. -->
-    <InstallerVersion Condition="'$(InstallerVersion)'==''">0.1</InstallerVersion>
-    <!-- default msi filename if not specified -->
-    <OutputName Condition="'$(OutputName)'==''">datadog-dotnet-apm-$(InstallerVersion)-$(Platform)-$(Configuration)</OutputName>
+    <InstallerVersion>1.9.1</InstallerVersion>
+    <OutputName>datadog-dotnet-apm-1.9.1-prerelease-$(Platform)-$(Configuration)</OutputName>
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|x86' ">
-    <DefineConstants>Debug;InstallerVersion=$(InstallerVersion)</DefineConstants>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|x86' ">
-    <DefineConstants>InstallerVersion=$(InstallerVersion)</DefineConstants>
+  <PropertyGroup>
+    <DefineConstants>InstallerVersion=1.9.1</DefineConstants>
```

Comment:
Do we need this? We can already read the `<InstallerVersion>` property above in the wxs file.

Reply:
Might be worth testing, but from first appearances, this is required to pass the value so the wxs file can access it from `$(var.InstallerVersion)`

Reply:
One separate optimization could be to set this to `InstallerVersion=$(InstallerVersion)` that way you don't have to manipulate this line at all, you'd only have to manipulate the above lines.

Reply:
Once it's set it's set and I don't care all too much, can we leave it in?

Reply:
This is the right one. It's the build property `<InstallerVersion>` that we don't need anymore. It was there so we could set the filename. I'll add a suggestion so you only need to click accept.